oak_containers_stage1/src/main.rs

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#![feature(never_type)]
+
+mod image;
+mod server;
+
+use clap::Parser;
+use std::{error::Error, net::SocketAddr};
+
+#[derive(Parser, Debug)]
+struct Args {
+    #[arg(required = true)]
+    addr: SocketAddr,
+
+    #[arg(default_value = "/sbin/init")]
+    init: String,
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
+    let args = Args::parse();
+
+    // We want to send back a meaningful answer to the caller that the RPC has been handled
```

Comment:
I tried to clarify "We/our" in the following comment, but I wasn't sure. Could you clarify or maybe we (you and I ;) ) discuss offline?